src/program_generator/pg_arguments.py

```diff
@@ -108,6 +108,23 @@ def _do_generate(self, opcode, count, op_count):
 
   def _generate_program_triplet(self, operation, op_count):
     opcode = operation['Mnemonic']
+
+    # the program triplet will be for the following number of measured OPCODEs
+    op_counts = [0, op_count, op_count * 2]
+
+    if opcode == "CREATE":
+      opcode_arity = 3
+      mem_allocation = "5f61080052"
+      initial_code_on_stack = "6c6460016001015f526005601bf360981b"
+      initial_code_in_mem = ""
+      for pre_offset in range(32):
+        offset = "%0.4X" % (32 * pre_offset)
+        initial_code_in_mem += "8061" + offset + "52"
+      prefix_code = mem_allocation + initial_code_on_stack + initial_code_in_mem
+      arg_sizes = [1, random.randint(1, 32), random.randint(1, 32)]
+      single_op_pushes = ["61%0.4X" % (arg_sizes[2] * 32), "61%0.4X" % ((arg_sizes[1] - 1) * 32), "5f"]
```

Comment:
This is not obvious.
Please leave a descriptive comment explaining structure of the full code for CREATE,
explain why there is `SHL` used, what's in memory, etc.

Reply:
code is changed a little to be more straight forward